tokio/tests/sync_mpsc.rs

```diff
@@ -15,30 +15,31 @@ trait AssertSend: Send {}
 impl AssertSend for mpsc::Sender<i32> {}
 impl AssertSend for mpsc::Receiver<i32> {}
 
-#[test]
```

Comment:
Are these tests still relevant? I am not sure why they were left commented out (probably my fault during a refactor).

Reply:
These were commented out while I was changing the MPSC implementation, which we decided not to do. I forgot to un-comment them.